econml/inference/_bootstrap.py

```diff
@@ -139,8 +179,16 @@ def __getattr__(self, name):
 
         def proxy(make_call, name, summary):
             def summarize_with(f):
-                results = np.array(Parallel(n_jobs=self._n_jobs, prefer='threads', verbose=self._verbose)(
-                    (f, (obj, name), {}) for obj in self._instances)), f(self._wrapped, name)
+                instance_results = []
+                obj = clone(self._wrapped, safe=False)
+                for i in range(self._n_bootstrap_samples):
+                    if self._only_final:
+                        obj._set_current_cloned_ortho_learner_model_final(i)
+                    else:
+                        obj = self._instances[i]
+                    instance_results.append(f(obj, name))
+                instance_results = np.array(instance_results)
+                results = instance_results, f(self._wrapped, name)

```

Comment:
I think it would be better to continue to use parallelism here if possible.

Reply:
Based on our discussion, it seems like keeping this in `_bootstrap.py` and using parallel processes or threads will not be possible. In the future, we would like to make this call in `_ortho_learner.py`, just as we do with fit.